vite build reports "fs" browser compatibility issues

When bundling for deployment I see the following warning in the console:

```
[plugin vite:resolve] Module "fs" has been externalized for browser compatibility, imported by "/node_modules/signaldb/dist/index.mjs". See https://vitejs.dev/guide/troubleshooting.html#module-externalized-for-browser-compatibility for more details.
```

- Angular CLI: 17.3.6
- Node: 20.17.0
- Package Manager: npm 10.8.2

```

import { Collection, createLocalStorageAdapter } from 'signaldb';
import angularReactivityAdapter from 'signaldb-plugin-angular';

export const InventoryItems = new Collection<
  InventorySelectItem & { id: string }
>({
  reactivity: angularReactivityAdapter,
  persistence: createLocalStorageAdapter('localData'),
});

```

My code is above but I got this error and Angular app not build.

`./node_modules/signaldb/dist/index.mjs:2954:25-37 - Error: Module not found: Error: Can't resolve 'fs' in '/project_path/node_modules/signaldb/dist'`

Is there any solutions?

--- a/packages/signaldb/vite.config.ts
+++ b/packages/signaldb/vite.config.ts
@@ -30,10 +30,19 @@ export default defineConfig({
       name: 'SignalDB',
       entry: path.resolve(__dirname, 'src/index.ts'),
       fileName: format => (format === 'es' ? 'index.mjs' : `index.${format}.js`),
+      formats: ['es', 'cjs'],
     },
     rollupOptions: {
+      output: {
+        inlineDynamicImports: false,
+        preserveModules: true,
+        format: 'es',
+      },
       external: [
+        'fast-sort',
         'fs',
+        'mingo',
+        'mingo/updater',
       ],
       plugins: [
         typescriptPaths({

